paper-ieee-trans.tex

```diff
@@ -66,12 +66,12 @@ \section{Introduction}
 supporting an unified access to large-scale computing and intense data analysis
 applications, has been driven by the needs of scientific communities as part of their
 pan-European research collaborations. One such example is the Worldwide
-Large Hadron Collider (LHC) Computing Grid infrastructure, motivated
+Large Hadron Collider (LHC) Computing Grid infrastructure, driven
 by the particle physics scientific community, built to satisfy the computing needs
 of processing, storing and analysis of the very large amount of data produced by the
 LHC particle accelerator. A significant number of innovation projects and
-technology transfer activities -- from the 5th, 6th and 7th Framework Programmes,
-and recently by the Horizon 2020 programme \cite{h2020} -- are under way to
+technology transfer activities -- from the European Union's funded 5th, 6th and 7th Framework Programmes,
```

Comment:
I would use European Commission since is the term we use throughout the text. 

If we put this here we have to include the acronym since it is the first appearance --> "(..) from the European Commission (EC)-funded 5th, (..)" and remove it from where it is now

Reply:
agree with change
above I though was EC

Reply:
I agree with the use Of European Commission

Reply:
ok